sdk/ai/azure-ai-projects/samples/agents/sample_agents_morningstar.py

```diff
@@ -0,0 +1,193 @@
+# pylint: disable=line-too-long,useless-suppression
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with the
+    Morningstar OpenAPI tool from the Azure Agents service using a synchronous client.
```

Comment:
Maybe add document for Morningstar, to provide user more information what it is?

Reply:
Will work with our editorial team to have a customer facing page focused on this use case. For now, I've provided the link to the Morningstar website.